src/collections/members/anirudh-jain/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Anirudh Jain
+position: Contributer
+image_path: ./anirudh-jain.jpg
+github: anirudhjain75
+twitter: anirudhjain75
+linkedin: anirudh-jain-a19b5a104
+blurb: Mesh Head
+status: Active
+badges:
```

Comment:
Please follow the ordering of times as mentioned in the template (https://github.com/layer5io/layer5/blob/3f738b30b76e0d618910f279572157d5d82738d7/src/collections/members/_member-profile-template/index.mdx).

Reply:
You don't need to add empty fields.